packages/gatsby/src/internal-plugins/query-runner/file-parser.js

```diff
@@ -70,8 +70,7 @@ async function parseToAst(filePath, fileStr) {
         `There was a problem parsing "${filePath}"; any GraphQL ` +
           `fragments or queries in this file were not processed. \n` +
           `This may indicate a syntax error in the code, or it may be a file type ` +
-          `That Gatsby does not know how to parse.`,
-        error
```

Comment:
@jquense this wasn't supposed to be there right? The stack trace was just a long bunch of babel parsing nonsense.